### ISSUE TITLE
Why is the browser stuck when visualizing the generated surface mesh?

### ISSUE THREAD

**Issue by BoyuLyu** [seung-lab/igneous#160]
I have been trying to visualize a large volume, with 2000x2000x1000 voxels. I created a configuration file using `CloudVolume.create_new_info`, which I set the chunk to be 100x100x50 voxels. And then I use the following code to create downsampled data and visualizing in neuroglancer. However, when I tried to visualize the mesh in the neuroglancer, it become very laggy, which is almost impossible to interact with. I am wondering if there were ways to make it faster for visulization? I tried downsample the image with `--mip-level 4` added, but still it was not helping.
```
igneous image downsample file://./ --mip 0 --queue ./ds-queue 
igneous --parallel 20 execute -x ./ds-queue
igneous mesh forge file://./ --mip 0 --queue ./mesh-queue 
igneous --parallel 20 execute -x ./mesh-queue
igneous mesh merge file://./ --queue ./mesh-queue 
igneous --parallel 20 execute -x ./mesh-queue
igneous view ./ --port 6220
```

**Comment by william-silversmith** [comment]
I think i see what the problem is. Try meshing from a higher mip level,
that will radically simplify the meshes.

Usually I try to pick a near isotropic mip level.

**Comment by HHHit** [comment]
Thank you. Now, with the mip-level 4 as the input for generating the mesh, I can visualize in the browser more smoothly. However, the quality is really bad due to the low resolution. I am wondering if I can generate more levels of mesh? so that when I zoom in, the finer details of the mesh will be displayed.

**Comment by william-silversmith** [comment]
Yes, you can do that by generating the sharded multi-resolution meshes.

**Comment by HHHit** [comment]
Is the parameter nlod used to decide how many levels of mesh to generate?

**Comment by william-silversmith** [comment]
Yes, it sets the max number to generate though not the precise number.

**Comment by HHHit** [comment]
Thanks a lot! I have tried using the shared format, but I still found it only displays the mesh with only one resolution. Are there any issues in my following command? Thanks,
```
igneous image downsample file://./ --mip 0 --num-mips 4 --queue ./ds-queue
igneous --parallel 20 execute -x ./ds-queue/
igneous mesh forge file://./ --mip 4 --queue ./mesh-queue --sharded 
igneous --parallel 20 execute -x ./mesh-queue
igneous mesh merge-sharded file://./ --queue ./mesh-queue --nlod 4
igneous --parallel 20 execute -x ./mesh-queue  
igneous view ./ --port 6220
```

**Comment by william-silversmith** [comment]
I think you'll want to forge from a lower mip level for meshing if you want the multi-res stuff to be usable. The mip level sets the floor on the resolution you can obtain.
